package.json

```diff
@@ -18,7 +18,8 @@
     "copy-templates": "rsync -av --exclude '*node_modules*' ./src/code-templates/ ./.dist/src/code-templates",
     "build:watch": "tsc --watch",
     "start:cli": "node ./.dist/bin/cli.js interactive",
-    "publish:build": "npm run build && npm publish ./.dist --access public"
+    "publish:build": "npm run build && npm publish ./.dist --access public",
+    "prepare": "cd .. && husky install front/.husky"
```

Comment:
@rluvaton 
Thank you for pointing that out :)
In the CI process, `npm ci` completed successfully: 
<img width="444" alt="image" src="https://user-images.githubusercontent.com/58806763/179304949-5405b751-67b1-425c-b170-05015ff66870.png">

but when the tests were run, the process failed:
<img width="794" alt="image" src="https://user-images.githubusercontent.com/58806763/179305663-f18fe472-3062-446f-8535-9b02d85cc72b.png">

I followed that issue solutions [here](https://github.com/typicode/husky/issues/851) but it still fails.
Any idea what else can cause that?

Reply:
I know how to fix this, I'll create a PR, we shouldn't only publish the dist folder

Reply:
I won't be able to create the PR at the moment so maybe copy the husky script in build for now until I fix this issue

Reply:
@rluvaton Can you share your thoughts: What is the failure reason, what would you fix?

@idobetesh Does it fail locally or only in CI? Should indeed install huskey in CI?

Reply:
> Can you share your thoughts: What is the failure reason, and what would you fix?

the failure reason is that the `.git` folder is missing in the `.dist` folder, husky doesn't think to go one folder up as the `.dist` folder contains the `package.json` file


> Does it fail locally or only in CI

this issue happen both on local and on the CI

> Should indeed install huskey in CI?

There are valid use cases adding husky to the CI:
e.g. part of publishing a package we could generate a `CHANGELOG` file and commit + push it

Reply:
About the failure - Publishing to root won't necessary solve it, users who generate code might not have (yet) a git repo. I suggest that the optional solution is thoughtful Husky install command that fails if no Git exists. Can you find one like this please @rluvaton ? Other ideas?

About the need in CI - The idea of Husky is to prevent human mistakes, I don't see why a robot (CI) that publishes docs need another (robot) to check that it didn't forget to run tests